src/abstractdataframe/io.jl

```diff
@@ -134,11 +139,23 @@ function _show(io::IO, ::MIME"text/html", df::AbstractDataFrame;
         end
         for column_name in cnames
             if isassigned(df[!, column_name], row)
-                cell = sprint(ourshow, df[row, column_name])
+                cell_val = df[row, column_name]
+                if ismissing(cell_val)
+                    write(io, "<td><em>missing</em></td>")
```

Comment:
I Jupyter every second row is printed grey (see the picture below) so I think it would be problematic.

![Screenshot from 2020-04-22 10-43-35](https://user-images.githubusercontent.com/6187170/79960870-3a7d5d80-8486-11ea-83be-766809ce3520.png)